charts/roman/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for roman.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+roman:
+  replicaCount: 1
+  image:
+    repository: quay.io/wire/roman
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: staging
+  serviceToken: # Provide the roman service token.
+  # randomly generated for local testing
+  appKey: b53181dd-6400-4960-8988-f775545588ff-0949f503-421e-4588-a2c5-f64fd9c180fd
```

Comment:
I would comment out most of these variables. It's useful for people installing the chart to have an idea of what an example value looks like; but it's not a good idea if people mistakenly use these default values as they forget to override them. It would be better to have the chart fail to install if a variable is missing.